### ISSUE TITLE
Implement a way to allow applications to fake/mock the `roChannelStore` response/data

### ISSUE THREAD

**Issue by lvcabral** [lvcabral/brs-engine#138]

**Comment by upintheairsheep** [comment]
I'm glad you are working on the project again!
Also, why are these out of scope?
RSG (Roku SceneGraph) SDK support.
Video playback or streaming.
SDK 1.0 deprecated components.

**Comment by lvcabral** [comment]
> I'm glad you are working on the project again! Also, why are these out of scope? RSG (Roku SceneGraph) SDK support. Video playback or streaming. SDK 1.0 deprecated components.

Yep, working hard for the last couple of months, very soon I will release v0.10 that will come with huge performance improvements!! And I will remove RSG and Video Playback from the out of scope list, not sure when and how, but I will try to add those too. The SDK 1.0 I don't think worth the effort to do.

**Comment by upintheairsheep** [comment]
> > I'm glad you are working on the project again! Also, why are these out of scope? RSG (Roku SceneGraph) SDK support. Video playback or streaming. SDK 1.0 deprecated components.
> 
> Yep, working hard for the last couple of months, very soon I will release v0.10 that will come with huge performance improvements!! And I will remove RSG and Video Playback from the out of scope list, not sure when and how, but I will try to add those too. The SDK 1.0 I don't think worth the effort to do.

The SDK 1.0 should be in scope for the community to do, or after everything else is done.
